sandbox/apparmor/notify/strings.go

```diff
@@ -39,6 +42,69 @@ func (sp *stringPacker) PackString(s string) uint32 {
 	return offset + uint32(sp.baseOffset)
 }
 
+// packTagsets computes the layout of the tagsets encoded in an apparmor message.
```

Comment:
this doc is a bit confusing as the function actually also packs the tagsets, also it doesn't clearly describe how ts is organized in the input, are the keys permissions?